src/client/results/MostUsedWords.js

```diff
@@ -89,6 +79,28 @@ const WordItem = ({ wordInfo }) => (
   </div>
 );
 
+let WordItem = ({ dispatch, makeupInfo, conflatedWordCount }) => (
+  <tr>
+    <td className='right aligned' style={{ width: '60px' }}>{Math.round(makeupInfo.count / conflatedWordCount * 100)}%</td>
+    <td className='right aligned' style={{ width: '60px' }}>{makeupInfo.count}</td>
+    <td>{makeupInfo.word}</td>
+    <td>
+      <div className="ui checkbox" onClick={(e) => {
+        const isNowChecked = $($(e.target).parent().children()[0]).is(':checked');

```

Comment:
use react refs:

-add a ref node to the input and replace $(e.target).parent().children()[0] with the ref name since that just looks like some black magic :S